database-plugins/widgets/Database-batchsource.json

```diff
@@ -52,6 +52,21 @@
             ],
             "default": "false"
           }
+        },
+        {
+          "widget-type": "select",
+          "label": "Database Isolation Level",
+          "name": "databaseIsolationLevel",
+          "widget-attributes": {
+            "values": [
+              "TRANSACTION_NONE",
```

Comment:
It's all selected from a dropdown menu so the user shouldn't have to enter anything manually.